tests/test_ci/TestControllers.py

```diff
@@ -183,25 +183,64 @@ def test_customizedtest_added_to_queue(self):
+                     Some random string(https://github.com/Some random string)\n\n\n
+                     Lorem Ipsum sit dolor amet...\n        """
         email = inform_mailing_list(mock_email, "matejmecka", "2430", "Some random string",
                                     "Lorem Ipsum sit dolor amet...")
 
         mock_email.send_simple_message.assert_called_once_with(
             {
-                'text': '2430 - Some random string\n\n'
-                '        Link to Issue: https://www.github.com/test_owner/test_repo/issues/matejmecka\n\n'
-                '        Some random string(https://github.com/Some random string)\n\n\n'
-                '        Lorem Ipsum sit dolor amet...\n        ',
-                'subject': 'GitHub Issue #matejmecka', 'to': 'ccextractor-dev@googlegroups.com'
+                'to': 'ccextractor-dev@googlegroups.com',
+                'subject': 'GitHub Issue #matejmecka',
+                'html': """2430 - Some random string\n\n
+                     Link to Issue: https://www.github.com/test_owner/test_repo/issues/matejmecka\n\n
+                     Some random string(https://github.com/Some random string)\n\n\n
+                     Lorem Ipsum sit dolor amet...\n        """
             }
         )
+        mock_get_html_issue_body.assert_called_once()
+
+    @staticmethod
+    def test_get_html_issue_body():
+        """
+        Test the get_html_issue_body for correct email formatting
+        """
+        from mod_ci.controllers import get_html_issue_body
+
+        title = "[BUG] Test Title"
+        author = "abcxyz"
+        body = ("**strong_text**<br/>*em_text*\n\n"
+                "- [list_with_link](www.example.com)\n"
+                "- [X] checkbox_list with_code\n\n"
+                "`code`\n\n"
+                "> quote")
+        issue_number = 1
+        url = "www.example.com"
+
+        expected_html_issue_body = ("[BUG] Test Title - abcxyz<br/>\n"
+                                    "Link to Issue: www.example.com<br/>\n"
+                                    """<a href="https://github.com/abcxyz">abcxyz</a><br/><br/>\n"""
+                                    "<p><strong>strong_text</strong><br/><em>em_text</em></p>\n\n"
+                                    "<ul>\n"
+                                    """<li><a target="_blank" href="www.example.com">list_with_link</a></li>\n"""
+                                    "<li>[X] checkbox_list with_code</li>\n"
+                                    "</ul>\n\n"
+                                    "<p><code>code</code></p>\n\n"
+                                    "<blockquote>\n"
+                                    "  <p>quote</p>\n"
+                                    "</blockquote>\n")
+        received_html_issue_body = get_html_issue_body(title, author, body, issue_number, url)
+
+        assert received_html_issue_body == expected_html_issue_body, "wrong issue html [formatted] email returned"
```

Comment:
Please don't use assert, rather use the self.assertEqual & other available assert function calls that are available with the testing framework :)

Reply:
I would do that, but may I know the reason just out of curiosity? AFAIK using assert makes the tests framework independent and is a better practice (as per the internet) because it allows ease of changing framework.

Reply:
I guess the reason is consistency. Although I did a quick search, and it seems that using "assert" is natively supported by other testing frameworks... Let's keep it then this way :)